opl-secret-ballot/Makefile

```diff
@@ -0,0 +1,59 @@
+ROOT_DIR := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+
+# Runs a local ephemeral docker instance, with current dir bind-mounted to /src
+DOCKER=docker run --rm -it --hostname=opl -u `id -u`:`id -g` -e HISTFILESIZE=0 -e HOME=/src --network=host -w /src -v $(ROOT_DIR):/src node:lts
+
+DOCKER_PNPM=$(DOCKER) /src/.local/share/pnpm/pnpm
+
+PNPM=pnpm
+
+all:
+	@echo If you\'re not using docker, run:
+	@echo
+	@echo  $$ make pnpm-install
+	@echo  $$ make build
+	@echo
+	@echo To install PNPM + package.json dependencies inside a container, run:
+	@echo
+	@echo 	$$ make docker-pnpm-install
+	@echo
+	@echo Then to open a shell inside the container:
+	@echo
+	@echo   $$ make docker-shell
+	@echo
+
+pnpm-install:
+	$(PNPM) install
+
+docker-pnpm-install:
```

Comment:
This rule should be about installing pnpm tool inside a docker and not running `pnpm install` to download deps right? Let's download those later as a prerequirement for building backend and frontend.